star-rx72n-firmware/lib/rx_hal/src/rx_cmt.c

```
@@ -167,6 +168,27 @@ internal_calculate_cmt_params(uint32_t frequency_hz, uint8_t* divider, uint16_t*
   return k_rx_err_invalid_arg;
 }
 
+/**
+ * @brief Configure CMT interrupt routing and priority
+ *
+ * @param[in] channel CMT channel
+ * @param[in] priority Interrupt priority
+ *
+ * @return k_rx_ok on success, error code on failure
+ */
+static rx_err_t internal_configure_cmt_interrupt(rx_cmt_channel_t channel, uint8_t priority)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_configure_cmt_interrupt' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)